packages/interface-internal/src/registrar.ts

```diff
@@ -21,46 +26,75 @@ export type {
    */
   StreamHandlerRecord
 }
+/**
+ * The `Registrar` module provides an interface for managing protocol handlers
+ * and topologies in a libp2p network. It enables registering and managing
+ * protocol-specific handlers, ensuring efficient peer-to-peer communication.
+ */
+/**
+ * The `Registrar` interface allows modules to register, manage, and remove
+ * protocol handlers and topology discovery mechanisms in libp2p.
+ */
 
```

Comment:
Must be a single comment, not two.

Also we can be a bit more specific in the comment:

```suggestion
/**
 * The `Registrar` provides an interface for registering protocol handlers -
 * these are invoked when remote peers open streams on the local node with the
 * corresponding protocol name.
 *
 * It also allows configuring network topologies for a given protocol(s). The
 * topology callbacks are invoked when a peer that supports those protocols
 * connects or disconnects.
 *
 * The Identify protocol must be configured on the current node for topologies
 * to function.
 */
```